emukit/bayesian_optimization/acquisitions/entropy_search.py

```diff
@@ -214,3 +215,70 @@ def _innovations(self, x: np.ndarray) -> tuple:
     def has_gradients(self) -> bool:
         """Returns that this acquisition has gradients"""
         return False
+
+
+class MultiInformationSourceEntropySearch(EntropySearch):
+    """
+    Entropy search acquisition for multi-information source problems where the objective function is the output of one
+    of the information sources. The other information sources provide auxiliary information about the objective function
+    """
+    def __init__(self, model: Union[IModel, IEntropySearchModel], space: ParameterSpace,
```

Comment:
Any chance you could move part of this __init__ in a separate method, to shorten it a bit? Maybe identifying info source param piece? Maybe creation of proposal function?

Reply:
Done